azure-pipelines/end-to-end-tests-dir/build-test-ports.ps1

```diff
@@ -22,3 +20,8 @@ if ($IsWindows) {
     Run-Vcpkg @commonArgs --overlay-ports="$PSScriptRoot/../e2e_ports" install vcpkg-find-acquire-program
     Throw-IfFailed
 }
+
+Run-Vcpkg @commonArgs --overlay-ports="$PSScriptRoot/../e2e_ports/overlays" install broken-no-name
+Throw-IfNotFailed
+Run-Vcpkg @commonArgs --overlay-ports="$PSScriptRoot/../e2e_ports/overlays" install broken-no-version

```

Comment:
These should check that the output matches "missing field".

This would diagnose the overlay being ignored as well as failing to output useful error information.